docs/contributing/releasing.md

```diff
@@ -0,0 +1,58 @@
+## Releasing
```

Comment:
I'm wondering if we can drop this file entirely, as contributors normally wouldn't publish tokio to crates.io

Reply:
I don't see any reason to keep this file as well for contributors, I've removed it